mlflow/entities/model_registry/model_version.py

```diff
@@ -83,6 +85,12 @@ def status_message(self):
         """String. Descriptive message for error status conditions."""
         return self._status_message
 
+    @property
+    def tags(self):
```

Comment:
@zhidongqu-db For consistency with Run Tags and Experiment Tags, this should be a dictionary of key:value pairs, rather than a list. See https://github.com/mlflow/mlflow/blob/master/mlflow/entities/run_data.py#L42